src/main/resources/application.properties

```diff
@@ -0,0 +1,11 @@
+# datasource PostgreSQl
+spring.sql.init.platform=postgres
+spring.datasource.url=jdbc:postgresql://localhost:5432/techItEasyModel
+spring.datasource.username=postgres
+spring.datasource.password=307000
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.jpa.database=postgresql
+spring.jpa.generate-ddl=true
+spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
+
```

Comment:
In de aandachtspunten staat het volgende: `In de application.properties staat spring.sql.init.mode=always i.p.v. spring.datasource.initialization-mode=always`. 

Dit heḃ jij volgens mij niet staan. Klopt het dat je je programma nog niet kan runnen? Of lukt dit wel?